src/components/Theme/style.css

```diff
@@ -61,40 +63,40 @@
   }
 
   /* hide the footer if view height is too small */
-  @media (max-height: 420px) {
+  @media (max-height: 420*@unit) {
```

Comment:
I think this is a hack introduce to make sure that the footer does not override the content when the window is shrank vertically (ie on the capture confirm screen). This line is causing the footer to disappear when the page is zoomed at browser level. This is probably out of scope but we should fix it to make sure that the footer does not disappear as well as fixing the real issues of the footer overriding the content when window is shrank.

Reply:
Discussed offline, but this is an issue, but not one really specific to this PR
I've [raised this internal ticket](https://jira.onfido.co.uk/browse/CX-3011) to fix up this issue in a separate effort

Reply:
but this one we want to keep as pixels for now, no?

Reply:
I was just talking to @stephencookdev . The hack was also incorrect, it was kicking in in desktop mode as well, because it's only looking at height. It should have a query for width as well. However, it is outside of the scope of this task.

For this task we should just revert it to `px` to be consistent with the other queries.